packages/graphql-api/src/modules/accounts-password/resolvers/mutation.ts

```diff
@@ -16,7 +16,12 @@ export const Mutation: MutationResolvers<ModuleContext<AccountsModuleContext>> =
   },
   createUser: async (_, { user }, { injector }) => {
     const userId = await injector.get(AccountsPassword).createUser(user as PasswordCreateUserType);
-    return injector.get(AccountsServer).options.ambiguousErrorMessages ? null : userId;
+    const noErrors = injector.get(AccountsServer).options.ambiguousErrorMessages;
```

Comment:
after looking trough the source `ambiguousErrorMessages` is not what I expected it to be :sweat_smile: 

As i get it it's just a flag to disguise error messages?

Could someone explain the current behavior to me? As I get it the implementation here is not actually related to any error, but instead just hides the registration success when `ambiguousErrorMessages` is set to true. Why?
```js
return injector.get(AccountsServer).options.ambiguousErrorMessages ? null : userId;
```

Reply:
Hey @sakulstra, there's an explanation of the reasoning behind this behavior here https://github.com/accounts-js/accounts/pull/525#issuecomment-447271280 

> The createUser function should fail silently if the email we try to create already exist, so no id returned. If I am an attacker and I want to discover which user registered to your service I will call the createUser endpoint with a random email. If I get the id back means the email does not exist in the system, but if the function return null means the user exist.
If we never return the id to the client an attacker will not be able to guess if the user exist or not.

Reply:
@TimMikeladze thanks for giving me the context :+1:
I tested it in it's current form and if i'm not misunderstanding the purpose it just doesn't work atm.

We `could` probably fix it by just wrapping
```
    const userId = await injector.get(AccountsPassword).createUser(user as PasswordCreateUserType);
```
in a `try ... catch` and only forward the error if `ambiguousErrorMessages` set to false, but i'd personally prefer just dropping it.